src/common/function/src/scalars/timestamp.rs

```diff
@@ -1,28 +0,0 @@
-// Copyright 2023 Greptime Team
```

Comment:
We can keep `TimestampFunction` even though there are no functions after this PR. But we will add more in future.

Reply:
Yep, the follow-up task `to_unixtime()` may need this. We can keep this file for now.

Reply:
sure, I've restored the scaffolding, could you please have another look?